src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -0,0 +1,978 @@
+﻿//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
```

Comment:
change to .net foundation header

Reply:
done

---
In reply to: [196948540](https://github.com/dotnet/machinelearning/pull/377#discussion_r196948540) [](ancestors = 196948540)

Reply:
#Resolved